packages/uswds-fonts/README.md

```diff
@@ -0,0 +1,47 @@
+# USWDS Fonts
+
+The component package for USWDS fonts. See font documentation on the [typesetting tokens page](https://designsystem.digital.gov/design-tokens/typesetting/overview/).
+
+## Features
+
+- Fonts
+  - merriweather
+  - public sans
+  - roboto mono
+  - source sans pro
+
+See `uswds-fonts/package.json` for entrypoints.
+
+## Getting started
+
+```bash
+npm install @uswds/uswds-fonts
+```
+
+## Running tests
+
+No tests available at this time.
+
+## Usage
+
+Use SASS to generate the font face rules.
+
+### SASS
+
+Make sure `node_modules/@uswds` is included in your paths.
+
+```scss
+  @forward "uswds-fonts";
```

Comment:
I noticed in the test react app that you included a custom config of the font path setting. Wondering if it is worth mentioning that setting in the README writeup in case users are new/need to update. 
```
@use "uswds-fonts" as * with (
  $theme-font-path: "~@uswds/uswds-fonts/src/assets/fonts"
);
```